packages/create-bison-app/template/_templates/graphql/new/graphql.ejs

```diff
@@ -3,20 +3,28 @@ to: graphql/modules/<%= name %>.ts
 ---
 <% camelized = h.inflection.camelize(name) -%>
 <% plural = h.inflection.pluralize(camelized) -%>
-import { objectType, extendType, inputObjectType, stringArg, arg, nonNull, enumType } from 'nexus';
-import { Role } from '@prisma/client';
-import { UserInputError, /*ForbiddenError*/ } from 'apollo-server-micro';
+import { objectType, extendType, inputObjectType, list, arg, nonNull /*, enumType*/ } from 'nexus';
 
-// import { isAdmin } from '../services/permissions';
+import { isAdmin } from '../../services/permissions';
+
+// Example ENUM Type
+// import { Role } from '@prisma/client';
+// export const UserRole = enumType({
+//   name: 'Role',
+//   members: Object.values(Role),
+// });
```

Comment:
it feels like we might delete this most of the time. I definitely see the value in outlining the `Object.values` example, but I did want to ask if you think this falls into the "you'll need this often" category.

Reply:
Agreed. 
It is not an everyday thing, probably something better suited for Docs over a generated file. 
Will opt to remove here.